### DISCUSSION TITLE
Review of Security Self Assessment Document of CubeFS

### DISCUSSION THREAD

**Discussion by avinashnarasimhan18** [cubefs/cubefs#2825] [General]
Hi everyone,

This is Avinash Narasimhan. I am a graduate student at NYU pursuing my Masters in Cybersecurity. As part of one of the projects I am doing as part of my course this semester, I am working on creating a Security Self Assessment Document for CubeFS. Me and my team have created an initial draft of the assessment. (Link to the Self Assessment Document - https://github.com/avinashnarasimhan18/tag-security/blob/main/assessments/projects/cubefs/self-assessment.md)

The next part of my project requires me to get that document reviewed by CubeFS project maintainers. If there is anyone who could us in getting the assessment reviewed or could help me connect with the right people, I would appreciate it.

**Comment by JustinCappos** [comment]
TAG Security Tech lead weighing in here.  We're planning to merge this assessment into the CNCF's TAG Security repo to provide security guidance to users about your project.  https://github.com/cncf/tag-security/tree/main/assessments

If you have any feedback about this assessment please follow up with the team above.  If you have questions about the process, please reach out to me, one of the TAG Security Chairs, or Chris Aniszczyk.
